### PR TITLE
Added about.html files to the binary build.

### LINKED ISSUE #2178: About.html files not included in the build.properties file
While working on a check for the static-analysis-tool that verifies the content of the build.properties file, I have noticed that in the most of the bundles the about.html file is not included in the build.properties file.

I assume that we would like to include the about.html file in the binary build.

I will extend the AboutHtmlCheck to verify that the about.html file is included in the build.properties file and create a PR that will fix the current code base in the OH2 addons repo.

@kaikreuzer, please let me know if it is not expected always to include the about.html file in the build.properties file.

### PR DESCRIPTION
**Pull request by svilenvul**
Closes #2178

Signed-off-by: Svilen Valkanov <svilen.valkanov@musala.com>

### CODE REVIEW

**Comment by svilenvul** [comment]
See also : https://github.com/openhab/static-code-analysis/pull/66

File: addons/binding/org.openhab.binding.feed/build.properties [hunk 1]
```diff
@@ -3,4 +3,5 @@ output.. = target/classes
 bin.includes = META-INF/,\
                .,\
                OSGI-INF/,\
-               ESH-INF/,\
\ No newline at end of file
+               ESH-INF/,\
+               about.html
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.feed/build.properties]: Missing newline

File: addons/binding/org.openhab.binding.max.test/build.properties [hunk 2]
```diff
@@ -3,4 +3,6 @@ source.. = src/test/groovy,\
 output.. = target/classes/
 bin.includes = META-INF/,\
                .,\
-               src/
+               src/,\
+               build.properties,\
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.max.test/build.properties]: why do you include the `build.properties`

File: addons/binding/org.openhab.binding.zway/build.properties [hunk 3]
```diff
@@ -5,5 +5,6 @@ bin.includes = META-INF/,\
                OSGI-INF/,\
                ESH-INF/,\
                lib/commons-lang3-3.4.jar,\
-               lib/zway-lib-0.1.7-SNAPSHOT.jar
+               lib/zway-lib-0.1.7-SNAPSHOT.jar,\
+               about.html
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.zway/build.properties]: Missing newline (or spaces on the last line)

**Comment by svilenvul** [comment]
>And one of them seems to include the build.properties now, is that planned?

Thanks @martinvw, I have removed it.

File: addons/binding/org.openhab.binding.feed/build.properties (hunk 1 above)
Comment (svilenvul) [inline addons/binding/org.openhab.binding.feed/build.properties]: It should be ok, now

**Comment by martinvw** [comment]
LGTM

**Comment by kaikreuzer** [comment]
Many thanks!

**Pull request merged**